### DISCUSSION TITLE
docs and cli page are not accessible any more

### DISCUSSION THREAD

**Discussion by drouxel** [salesforce/akita#824] [General]
hello,

I've been recommended to use Akita on my angular projects by a friend of mine and wanted to get to grips with it but the pages for documentation and cli are not accessible any more. ( [https://datorama.github.io/akita/](url) , [https://github.com/salesforce/akita/tree/master/tools/akita-cli](url)) is it a will or just a unfortunate problem

thanks for reading me

**Comment by NetanelBasal** [comment]
https://opensource.salesforce.com/akita/

**Reply by drouxel** [reply]
thanks a lot
